namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-development/resources/main.tf

```diff
@@ -1,22 +1,26 @@
 terraform {
   backend "s3" {}
 }
+
```

Comment:
Formatting for the `cica-development` resources has been fixed in master (94c6ff1fd00c7022cc347bc1d814003d7690ae73) - hopefully this won't create a merge conflict.